### ISSUE TITLE
Warning: Failed prop type: The prop `amPm` is marked as required in `Hour12Input`, but its value is `null`.

### ISSUE THREAD

**Issue by cab0** [wojtekmaj/react-datetime-picker#59]
react: 16.4.2
react-datetime-picker: 2.4.0

<DateTimePicker name={this.props.name} onChange={this.handleChange} value={this.state.setTime} calendarIcon={null} clearIcon={null} format='y-MM-dd h:mm'  />

Warning: Failed prop type: The prop `amPm` is marked as required in `Hour12Input`, but its value is `null`.

**Comment by mattheweberhardt** [comment]
i get the same
react 16.8.6
react-datetime-picker 2.4.0

**Comment by Nanyx** [comment]
I open up the source and if you add a null-validation on the prop "amPm" of the component <Hour12Input/> like soo :

```
<Hour12Input
        key="hour12"
        {...this.commonInputProps}
        amPm={amPm || "am"}
        showLeadingZeros={showLeadingZeros}
        value={hour}
/>
```

Solve the problem for my end. I use a `null` value for the default date. Anyhow, great job!

**Comment by wojtekmaj** [comment]
Hi,
this has been fixed with React-DateTime-Picker 2.5.0. Check it out!

**Comment by mattheweberhardt** [comment]
i can confirm that the error is now gone :-)
thank you for this project and your work!!
